src/firmware/main/app/world/firmware_robot.h

```diff
@@ -21,18 +48,35 @@ typedef struct FirmwareRobot FirmwareRobot_t;
  *                             robot, in meters
  * @param get_robot_position_y A function that can be called to get the y-position of the
  *                             robot, in meters
+ * @param get_robot_orientation A function that can be called to get the orientation of
```

Comment:
We should specify what coordinate frame all of this is in.

Reply:
I noted it at the top of this jdoc, but expanded on that:
```
 * NOTE: Everything in this class are in global field coordinates (ie. 0,0 is the 
 *       center of the field, 0 degrees is towards the enemy goal) unless 
 *       otherwise specified
```